### PR TITLE
chore: updates to site

### PR DESCRIPTION
**Pull request by hacktivist123**
- Adds new Nemlig, DigitalOcean, Sicredi, and Post Finance case studies to adopters page
- Adds link pool to adopters page
- Updates post finance case study link on the financial services page
- Adds nemlig to the e-commerce page
- Adds DigitalOcean to cloud providers and network policy page

### CODE REVIEW

File: src/utils/case-studies-data.js [hunk 1]
```diff
@@ -266,6 +271,11 @@ export default {
     iconName: 'digitalOcean',
     text: 'Managed Kubernetes: 1.5 Years of Cilium Usage at DigitalOcean',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:272]: can you update this description too?

File: src/utils/case-studies-data.js [hunk 2]
```diff
@@ -925,6 +940,21 @@ export default {
       },
     ],
   },
+  nemlig: {
+    iconName: 'nemlig',
+    text: '<b>Nemlig</b> uses Cilium in their Kubernetes clusters for networking, security and observability',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:945]:
```suggestion
    text: '<b>Nemlig</b> uses Cilium in their Kubernetes clusters for networking, security, and observability',
```

File: src/pages/industries/e-commerce.jsx [hunk 3]
```diff
@@ -116,17 +117,27 @@ const commerceResources = [
...
       'Trip.com looked at several options to pick its next-generation networking platform. Cilium provided them with an extremely scalable and stable networking solution.',
   },
+  {
+    imageSrc: NemligOfficeImage,
+    imageAlt: 'nemlig building',
+    title: 'Building a Kubernetes Platform with Cilium',
+    buttonText: 'Read The Case Study',
+    buttonLink: 'https://www.cncf.io/case-studies/nemlig/',
+    description:
+      'Implementing Cilium in their Kubernetes platform has enabled Nemlig to enhance agility, replace kube-proxy and iptables in their clusters, and reduce the operational burden of managing their network.',
```
Comment (xmulligan) [inline src/pages/industries/e-commerce.jsx:132]:
```suggestion
      'Implementing Cilium in their Kubernetes platform enabled Nemlig to enhance agility, replace kube-proxy and iptables in their clusters, and reduce the operational burden of managing their network.',
```

File: src/utils/case-studies-data.js (hunk 1 above)
Comment (hacktivist123) [reply src/utils/case-studies-data.js]: done

**Review by xmulligan — APPROVED** [review]

**Pull request merged**